apps/examples/testcase/le_tc/kernel/tc_libc_timer.c

```diff
@@ -415,6 +415,108 @@ static void tc_libc_timer_strftime(void)
 	TC_SUCCESS_RESULT();
 }
 
+/**
+* @fn             :tc_libc_timer_strptime
```

Comment:
can you please keep the test results in the commit description just for this test case alone ?

Reply:
ok

Reply:
I can't find the result of new tc in commit description. Will you leave or not?